nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -262,3 +262,5 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The option `services.nomad.extraSettingsPlugins` has been fixed to allow more than one plugin in the path.
 
 - The option `services.prometheus.exporters.pihole.interval` does not exist anymore and has been removed.
+
+- `nixos-rebuild` now uses `sudo` when deploying to localhost as a non-root user. This behaviour can be disabled with `--no-auto-sudo`.
```

Comment:
Please see the comment ``<!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
``